apph-front/src/services/UserService.ts

```diff
@@ -36,6 +36,34 @@ class UserService {
     };
     return Server.request(URL, requestOptions, successFunction, errorFunction);
   }
-}
 
-export default UserService;
+  static getUser(
+    handleSuccess: (user: string) => void,
+    handleError: (errorMessage: string) => void
+  ) {
+    const { login, ...token } = cookies.get('user');
+    const requestOptions = {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+        Authentication: token.token
+      }
+    };
+
+    const successFunction = (user: string) => {
+      if (user !== null) {
+        handleSuccess(user);
+      }
+    };
+    const errorFunction = (errorMessage: string) => {
+      handleError(errorMessage);
+    };
+
+    return Server.request(
+      `${API_URL}/user/${login}`,
+      requestOptions,
+      successFunction,
+      errorFunction
+    );
```

Comment:
Tu peux lui passer directement tes fonctions je pense
```suggestion
 return Server.request(
      `${API_URL}/user/${login}`,
      requestOptions,
      handleSuccess,
      handleError
    );
```